ddtrace/tracer/metrics.go

```diff
@@ -91,8 +91,8 @@ func (t *tracer) reportHealthMetrics(interval time.Duration) {
 	for {
 		select {
 		case <-ticker.C:
-			t.statsd.Count("datadog.tracer.spans_started", int64(atomic.SwapUint32(&t.spansStarted, 0)), nil, 1)
-			t.statsd.Count("datadog.tracer.spans_finished", int64(atomic.SwapUint32(&t.spansFinished, 0)), nil, 1)
+			t.statsd.Count("datadog.tracer.spans_started", int64(atomic.SwapUint32(&t.spansStarted, 0)), []string{"integration:manual"}, 1)
```

Comment:
Hmmm....

What's happening here is the spansStarted and spansFinished counters are being incremented somewhere else in the code (i.e, inside of span.Start and span.Finish). At the specified interval, we report the current value (count) to the statsd client. So, by the time we are in this code block, we're reporting about a "chunk" of spans started/spans finished. If we want to report the `integration`, this will be tied to the individual span started/finished. We will only know this at the time we increment the counters.

To report the `integration` tag, we will either have to:
1. Report these metrics on each invocation of span.Start / span.Finish, rather than inside of this goroutine (Cons: These operations happen frequently and this will be very noisy)
2. Change the way we track spansStarted and spansFinished to be some kind of counter map that includes the integration name, e.g. `map[string]uint32` where the key is the integration name and the value is the count of spans started/finished that integration name. Then, in this goroutine, we'll have to iterate over the map and report the spans started/finished per integration
3. Some other idea I haven't thought of?